src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -107,19 +201,37 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
                 caseData.setInterlocReviewState(null);
             }
         }
+        return buildResponse(callback, caseDetails, caseData, sscsCaseDataPreSubmitCallbackResponse, url, documentTranslationStatus);
+    }
+
+    private PreSubmitCallbackResponse<SscsCaseData> buildResponse(Callback<SscsCaseData> callback,
+                                                                  CaseDetails<SscsCaseData> caseDetails,
+                                                                  SscsCaseData caseData,
+                                                                  PreSubmitCallbackResponse<SscsCaseData> sscsCaseDataPreSubmitCallbackResponse,
+                                                                  DocumentLink url,
+                                                                  SscsDocumentTranslationStatus documentTranslationStatus) {
+
+        caseData = updateCaseForDirectionType(caseDetails, caseData);
+
 
         if (callback.getEvent() == EventType.DIRECTION_ISSUED) {
             footerService.createFooterAndAddDocToCase(url, caseData, DocumentType.DIRECTION_NOTICE,
-                Optional.ofNullable(caseData.getDateAdded()).orElse(LocalDate.now())
-                    .format(DateTimeFormatter.ofPattern("dd-MM-YYYY")),
-                caseData.getDateAdded(), null, documentTranslationStatus);
+                    Optional.ofNullable(caseData.getDateAdded()).orElse(LocalDate.now())
+                            .format(DateTimeFormatter.ofPattern("dd-MM-YYYY")),
+                    caseData.getDateAdded(), null, documentTranslationStatus);
```

Comment:
Formatting looks weird. Does adding the extra tab make it more readable?

Reply:
could not be more readable :)

![image](https://user-images.githubusercontent.com/4736395/92900742-d862cc00-f417-11ea-8b4b-13c0b34c9351.png)

Reply:
yes keeps it correctly aligned for reading